java/java.api.common/src/org/netbeans/modules/java/api/common/singlesourcefile/Description.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<!--
+    akshay
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
The URL would be right, but our check tool (apache creadur rat) does not support https yet. Please change the URLs to http://.

Reply:
Thank you @matthiasblaesing for all the above suggetions